editor/input_event_configuration_dialog.cpp

```diff
@@ -533,6 +533,9 @@ String InputEventConfigurationDialog::_get_device_string(int p_device) const {
 
 void InputEventConfigurationDialog::_notification(int p_what) {
 	switch (p_what) {
+		case NOTIFICATION_VISIBILITY_CHANGED: {
+			event_listener->grab_focus();
+		}
```

Comment:
Falls through but not marked implicit. Should either end with `break;` or `[[fallthrough]];`.

Reply:
Oops.